docs/contribute/dev-contribute.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-title: Contribute
+title: How to Contribute
 ---
 
 Thanks for contributing to Prefect!
```

Comment:
```suggestion
We welcome your contributions! 
Set up Prefect for development to experiment with extensions and improvements.
```

Reply:
Open to other things here but I think we can do better than what was there.

Reply:
@billpalombi i incorporated your change - I don't think it's the right use of my time right now vs. transactions but I want to revisit this page and have a top level section that is explicitly about making Pull Requests / opening Issues, as I think those fall under "Contributing", and I think that will fill out this page nicely.